### PR TITLE
Do not allow 16 bits through

### PR DESCRIPTION
**Pull request by GretaCB**
Mapnik will accept 16 bit TIFFs, but our stack does not support 16 bit yet.

cc @rclark

### CODE REVIEW

**Comment by rclark** [comment]
Awesome, thanks!

```js
metadata.raster.bands[0].rasterDatatype !== 'Byte'
```

This will be true for *every tiff other than an 8-bit one*? Should the error indicate instead that `Only 8-bit tiffs are supported`?

> Mapnik will accept 16 bit TIFFs

I don't think this is true. It won't *error* on them, but it also won't render them in any useful way.

File: index.js [hunk 1]
```diff
@@ -18,6 +18,10 @@ function Omnivore(uri, callback) {
 
   function getXml(err, metadata) {
     if (err) return callback(err);
+    
+    // Stopgap while only 8 bit TIFFs are supported
+    if (metadata.dstype === 'gdal' && metadata.raster.bands[0].rasterDatatype !== 'Byte') return callback('16 bit TIFFs are not supported');
```
Comment (rclark) [inline index.js]: If there is any scenario where `metadata.raster` is null, or `metadata.raster.bands` is empty or not an array, then this check could fail in an unhandled fashion. Should we check that everything that's expected back from omnivore is in place?

**Comment by GretaCB** [comment]
> Mapnik will accept 16 bit TIFFs
Meant to say that Mapnik will not error out on these (technically "accept" but won't render properly), so we need to error out on our end. 

Changing error message.

**Comment by GretaCB** [comment]
mapnik-omnivore PR to deal with zero bands scenario: https://github.com/mapbox/mapnik-omnivore/pull/69

metadata.raster can't be null for a TIFF since mapnik-omnivore will error out. And the (if 'gdal') check will ensure that the file is a TIFF.

**Comment by GretaCB** [comment]
Merged https://github.com/mapbox/tilelive-omnivore/pull/6

**Pull request merged**